docker/.env

```
@@ -143,12 +143,12 @@ SEEKDB_MEMORY_LIMIT=2G
 # WARNING: Change this for production!
 MYSQL_PASSWORD=infini_rag_flow
 # The hostname where the MySQL service is exposed
-MYSQL_HOST=mysql
+MYSQL_HOST=127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MYSQL_HOST key should go before the MYSQL_PASSWORD key

(UnorderedKey)